internal/ad/testdata/AD/SYSVOL/assetsonly.com/Distrolowercasegpt/GPT.ini

```diff
@@ -0,0 +1,3 @@
+[General]
```

Comment:
nickpitt on the folder name, but it’s actually the extension which is lowercase, not the whole name.

Reply:
:p